src/index.html

```diff
@@ -1,22 +1,474 @@
+                </a>
+              </li>
+
+            </ul>
+
+          </nav>
+
+          <a
+            href="tel:+1 234 5555-55-55"
+            class="menu__phone-number"
+          >
+            +1 234 5555-55-55
+          </a>
+
+          <a
+            href="tel:+1 234 5555-55-55"
+            class="menu__call-to-order"
+          >
+            call to order
+          </a>
+
+        </div>
+      </div>
+    </aside>
+
+    <main class="main">
+      <div class="container">
+        <div class="main__content">
+
+          <section class="recommended">
+
+            <h2 class="section__title">Recommended</h2>
+
+            <div class="recommended__products">
+
+              <article class="product recommended__product">
+                <img
+                  src="images/recommend_p1.svg"
+                  class="product__photo"
+                  alt="BEOSOUND A5"
+                />
+
+                <h3 class="product__title">BEOSOUND A5</h3>
+
+                <p class="product__category">
+                  Huge sound anywhere – in the home or outdoors. Beosound A5 is
+                  a Wi-Fi and Bluetooth speaker with long-lasting battery and
+                  wireless charging built in.
+                </p>
+
+                <p class="product__price">$ 1299.00</p>
+              </article>
+
+              <article class="product recommended__product">
+                <img
+                  src="images/recommend_p2.svg"
+                  class="product__photo"
+                  alt="BEOPLAY HX"
+                />
+
+                <h3 class="product__title">BEOPLAY HX</h3>
+
+                <p class="product__category">
+                  Wireless noise-cancelling headphones. Listen in comfort
+                  whether you're on a call, working from home or taking a moment
+                  for yourself.
+                </p>
+
+                <p class="product__price">$ 499.00</p>
+              </article>
+
+              <article class="product recommended__product">
+                <img
+                  src="images/recommend_p3.svg"
+                  class="product__photo"
+                  alt="BEOSOUND THEATRE"
+                />
+
+                <h3 class="product__title">BEOSOUND THEATRE</h3>
+                <p class="product__category">
+                  Four patent-pending acoustic innovations in one sleek soundbar
+                  envelop you in sound from every angle. Made to fit any TV, and
+                  last for generations.
+                </p>
+
+                <p class="product__price">$ 3199.00</p>
+              </article>
+            </div>
+          </section>
+
+          <section class="categories">
+            <h2 class="section__title">
+              Sound. Vision.
+              <br />
+              It’s your pick.
+            </h2>
+
+            <div class="categories__wrapper">
+              <article class="category">
```

Comment:
I don't see any interactivity on your cards. No `scale`, no `cursor: poiner`.
Fix it for all the cards